bindings/node/test/clientEncryption.test.js

```diff
@@ -77,7 +77,7 @@ describe('ClientEncryption', function () {
 
     beforeEach(function () {
       if (requirements.SKIP_LIVE_TESTS) {
-        this.test.skipReason = `requirements.SKIP_LIVE_TESTS=${requirements.SKIP_LIVE_TESTS}`;
+        this.currentTest.skipReason = `requirements.SKIP_LIVE_TESTS=${requirements.SKIP_LIVE_TESTS}`;
```

Comment:
to properly attach a skip reason, we need to attach it to the `currentTest`, not the `test` when in `before` blocks.